### DISCUSSION TITLE
is it possible to create multiple different graphs on single paper

### DISCUSSION THREAD

**Discussion by VADDIJASWANT** [clientIO/joint#2420] [Q&A]
### Introduction

Hi guys, I have a small doubt can we create multiple graphs on a single paper, if so can any one provide me relevant documents or examples which I can use as reference

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment]
Could you elaborate on what you mean by that? Isn't it related to: https://github.com/clientIO/joint/discussions/2171?
